docs/source/user-guide/general/ci_cd.rst

```diff
@@ -0,0 +1,130 @@
+=================================
+Using Deepchecks In CI/CD
+=================================
+
+CI/CD is a software engineering concept that is used to streamline the process of building, testing and deploying
+software products. CI/CD can also be utilized for the ML model lifecycle - to streamline the process of model training
+(and retraining), model validation and model deployment. This in turn reduces the risk of errors and improve the overall
+quality of the model.
+
+For example, CI/CD in machine learning can be used in different steps such as:
+
+* Data integrity validation: When the data used for training is collected via automatic processes and pipelines,
+  it is possible the data will contain errors and problems we haven't encountered before, either due to a bug in the
+  collection process or due to a change in the data source.
+  Examples of such problems include: missing values, outliers samples, high correlation between features,
```

Comment:
maybe put one of our cooler examples, such as conflicting labels and different spellings of categories.